yarpcconfig/configurator_test.go

```diff
@@ -476,7 +492,57 @@ func TestConfigurator(t *testing.T) {
 			},
 		},
 		{
-			desc: "implicit outbound unary and oneway",
+			desc: "implicit outbound stream",
+			test: func(t *testing.T, mockCtrl *gomock.Controller) (tt testCase) {
+				type transportConfig struct{ Address string }
+				type outboundConfig struct{ Queue string }
+				tt.serviceName = "foo"
+				tt.give = whitespace.Expand(`
+					outbounds:
+						bar:
+							redis:
+								queue: requests
+					transports:
+						redis:
+							address: localhost:6379
+				`)
+
+				redis := mockTransportSpecBuilder{
+					Name:                 "redis",
```

Comment:
Might be better to call this fake-streaming-transport, to avoid arguments about whether redis *could* be or *is* a multi-modal transport.